routes/_404.tsx

```diff
@@ -18,26 +16,26 @@ export default function Error404() {
       <div class="px-4 py-8 mx-auto">
         <div class="max-w-screen-md mx-auto flex flex-col items-center justify-center">
           <div class="flex flex-col justify-center items-center">
-            <div class={titleGreyLD}>WHOOPS</div>
```

Comment:
why was this changed from the prior updates babalicious made?

Reply:
yes, it was changed gray to purple from the latest update